promise/tests/test-promise.js

```diff
@@ -131,7 +131,7 @@ o.spec("promise", function() {
 				callAsync(function() {resolve(promise)})
 			})
 
-			promise.then(null, done)
+			promise.then(null, () => done())
```

Comment:
I mean, I have some other arrows in this PR. Should I convert them too? I assume yes.